README.md

```diff
@@ -13,15 +14,890 @@
 
 ---
 
-These libraries give you access to the powerful Parse Server backend from your Dart/Flutter app. For more information about Parse Platform and its features visit [parseplatform.org](https://parseplatform.org/).
+This library gives you access to the powerful Parse Server backend from your Dart app. For more information on Parse Platform and its features, visit [parseplatform.org](https://parseplatform.org). The Flutter package was moved [here](https://pub.dev/packages/parse_server_sdk_flutter). If you are using Flutter see [this guide](https://github.com/parse-community/Parse-SDK-Flutter/blob/release/2.0.0/docs/migrate-2-0-0.md) for how to migrate.
```

Comment:
I believe the migrate 2-0-0 document is deleted in this PR?

Would this release benefit from a new migration document?

Reply:
This version is related to Doc Dart, of course, the migration to version 2 section is no longer needed, I will add things for the new version.